<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(22, 45, 68);
border-radius:10px;</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>681</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>21</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 21pt &quot;MV Boli&quot;;
background-color: rgb(0, 170, 255);
text-decoration: underline;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TEXT RECOGNITION &amp; STORE IN DATABASE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="extracted_text">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>361</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>100</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Extracted Text</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_text_btn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>490</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: rgb(255, 255, 255);
 background-color: rgb(14, 139, 255);
border-radius:10px;
font: 10pt &quot;Arial Rounded MT Bold&quot;;
}

QPushButton:hover{
color: rgb(255, 255, 255);
	background-color: rgb(0, 170, 255);
font: 10pt &quot;Arial Rounded MT Bold&quot;;
}
</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
      <property name="icon">
       <iconset resource="../../../Qt Desiger/Icon.qrc">
        <normaloff>:/icon/Icon/save.svg</normaloff>:/icon/Icon/save.svg</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="change_img_btn">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>490</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: rgb(255, 255, 255);
 background-color: rgb(14, 139, 255);
border-radius:10px;
font: 10pt &quot;Arial Rounded MT Bold&quot;;
}

QPushButton:hover{
color: rgb(255, 255, 255);
	background-color: rgb(0, 170, 255);
font: 10pt &quot;Arial Rounded MT Bold&quot;;
}
</string>
      </property>
      <property name="text">
       <string>Change Image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="copy_text_btn">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>490</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: rgb(255, 255, 255);
 background-color: rgb(14, 139, 255);
border-radius:10px;
font: 10pt &quot;Arial Rounded MT Bold&quot;;
}

QPushButton:hover{
color: rgb(255, 255, 255);
	background-color: rgb(0, 170, 255);
font: 10pt &quot;Arial Rounded MT Bold&quot;;
}
</string>
      </property>
      <property name="text">
       <string>Copy</string>
      </property>
      <property name="icon">
       <iconset resource="../../../Qt Desiger/Icon.qrc">
        <normaloff>:/icon/Icon/copy.svg</normaloff>:/icon/Icon/copy.svg</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="image_box">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>130</y>
        <width>521</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="copied_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>455</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 170, 0);
background-color: transparent;
font: 9pt &quot;Arial Rounded MT Bold&quot;;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="error_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>450</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
font: 9pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(235, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>error_label</zorder>
     <zorder>copied_label</zorder>
     <zorder>label</zorder>
     <zorder>extracted_text</zorder>
     <zorder>label_5</zorder>
     <zorder>label_4</zorder>
     <zorder>save_text_btn</zorder>
     <zorder>change_img_btn</zorder>
     <zorder>copy_text_btn</zorder>
     <zorder>image_box</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Qt Desiger/Icon.qrc"/>
 </resources>
 <connections/>
</ui>
